src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -58,7 +58,7 @@
 
 ansi_dialect.set_lexer_matchers(
     [
-        RegexLexer("whitespace", r"[\t ]+", WhitespaceSegment),
+        RegexLexer("whitespace", r"[\t\xc2\xa0 ]+", WhitespaceSegment),
```

Comment:
@jpers36 I don't doubt this works but feels a bit inelegant. Doing a bit of research I saw a comment that the regex `\s` also matches non-breaking spaces, however it also matches newlines.

A good post [here](https://stackoverflow.com/questions/3469080/match-whitespace-but-not-newlines) suggests we can use this double negative regex pattern `[^\S\r\n]` to do `\s` but not newlines.
Could you try that please and see if that also solves the issue?

Reply:
@jpy-git that works, thanks!

Reply:
Nice!